esrally/telemetry.py

```diff
@@ -229,6 +229,8 @@ def instrument_java_opts(self):
 
     def java_opts(self, log_file):
         recording_template = self.telemetry_params.get("recording-template")
+        delay = self.telemetry_params.get("delay")
+        duration = self.telemetry_params.get("duration")
```

Comment:
I wonder if we should prefix these (and possible also the `recording-template`) with `jfr` i.e. `jfr-delay` and `jfr-duration`.
The rationale is that telemetry device parameters are not namespaced and the meaning of `delay` and `duration` is too wide to be dedicated only for the `jfr` telemetry device.

While at it, we could also rename `recording-template` but a) it'd be a breaking change for some and should also be documented in `migrate.rst` and b) `recording-template` is less generic and thus not so important to change right away.

WDYT?

Reply:
I agree that adding the `jfr` prefix would make it more consistent with the existing parameter naming convention.
If we are not adding it to the `recording-template` then wouldn't it break this consistency?

Reply:
I talked to @dliappis offline. It's OK to keep it like this for now. As PEP 8 says, a foolish consistency is the hobgoblin of little minds. :)

That said, in the future we'll want to switch. This requires a deprecation cycle, much like was done for relative-time (search for it in https://esrally.readthedocs.io/en/stable/migrate.html). Can you please create an issue about migrating to `jfr-recording-template` so that we don't forget to do it?

Reply:
Hey @pquentin, just to confirm the todos:

1. Add the prefix` jfr` to `delay` and `duration` making them `jfr-delay` and `jrf-duration` respectively.
2. Leave `recording-template` as is.
3. Create a separate issue for `jfr-recording-template`

Reply:
I've pushed the changes in the latest commit. Additionally https://github.com/elastic/rally/issues/1631 has been raised for the `jfr-recording-template` migration.